src/model/game/GameManager.ts

```diff
@@ -995,6 +995,12 @@ export default class GameManager {
             "https://thunderstore.io/c/painting-vr/api/v1/package-listing-index/",
             [new StorePlatformMetadata(StorePlatform.STEAM, "1905940")], "PaintingVr.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.RECURSIVE_MELON_LOADER, ["pvr"]),
+
+        new Game("DEPO : Death Epileptic Pixel Origins", "DEPO", "DEPO",
+            "DEPO", ["DEPO.exe"], "DEPO_Data",
+            "https://thunderstore.io/c/depo/api/v1/package-listing-index/",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "1091320")], "DEPO.png",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, ["depo"]),
```

Comment:
I don't think "depo" search string does anything here as depo is already in the title.